core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/project/ProjectResource.scala

```diff
@@ -236,7 +236,7 @@ class ProjectResource {
   def listProjectFiles(
       @PathParam("pid") pid: UInteger,
       @Auth user: SessionUser
-  ): List[DashboardFileEntry] = {
+  ): List[DashboardFile] = {
```

Comment:
Line 226 is an example of unnamed documentation.

Reply:
I have searched and fixed the documentation in all files.